src/Data/String/Unsafe.agda

```diff
@@ -21,3 +24,19 @@ toList∘fromList s = trustMe
 
 fromList∘toList : ∀ s → fromList (toList s) ≡ s
 fromList∘toList s = trustMe
+
+toList-++ : ∀ s t → toList (s ++ t) ≡ toList s List.++ toList t
+toList-++ s t = trustMe
+
+length-++ : ∀ s t → length (s ++ t) ≡ length s + length t
+length-++ s t = begin
+  length (s ++ t)                         ≡⟨⟩
+  List.length (toList (s ++ t))           ≡⟨ cong List.length (toList-++ s t) ⟩
+  List.length (toList s List.++ toList t) ≡⟨ Listₚ.length-++ (toList s) ⟩
+  length s + length t                     ∎ where open ≡-Reasoning
```

Comment:
Can we open `≡-Reasoning` at the top of the file rather than opening multiple times?